api/src/main/java/com/homeaway/datapullclient/process/DataPullTask.java

```diff
@@ -282,6 +283,20 @@ private RunJobFlowResult runTaskInNewCluster(final AmazonElasticMapReduce emr, f
                 .withClassification("emrfs-site")
                 .withProperties(emrfsProperties);
 
+        Map<String, String> sparkHiveProperties = emrProperties.getSparkHiveProperties().entrySet().stream().filter(keyVal -> !keyVal.getValue().isEmpty()).collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
+        sparkHiveProperties.putAll(this.clusterProperties.getSparkHiveProperties());
```

Comment:
putAll will take properties read from the input json and overwrite any default values already placed in sparkHiveProperties from application.yml.